.gitignore

```diff
@@ -7,4 +7,4 @@
 .externalNativeBuild
 .cxx
 local.properties
-/.idea
+**/.idea**
```

Comment:
Why the two stars?

Reply:
lol still trying to get this to work. I want it to never track anything in that package or anything that includes that name